### PR TITLE
Naming and logging stats

### PR DESCRIPTION
**Pull request by mihikabairathi**
Renamed variables to follow style guide, and added error and warning log stats to the output json string.

### CODE REVIEW

File: config_converter/config_mapper/config_mapper.py [hunk 1]
```diff
@@ -76,34 +76,39 @@ def extract_root_dirs(config_obj: config_pb2.Directive) -> tuple:
...
             logging.error('We do not know plugin %s', plugin_name)
         else:
-            plugin_dir = dir_name_map[d.name]
+            plugin_dir = dir_name_map[directive.name]
             if plugin_dir not in logs_module:
                 logs_module[plugin_dir] = []
             current_attribute_count = stats['attributes_recognized']
             logs_module[plugin_dir].append(
-                _convert_plugin(d, plugin_name, stats))  # stats are updated
+                _convert_plugin(directive, plugin_name, stats))  # update stats
```
Comment (StevenYCChou) [inline config_converter/config_mapper/config_mapper.py:109]: I think `stats are updated` is better - as this function not only update stats but also convert plugin. 

`stats are updated` is the part that users may not notice. What do you think?

File: config_converter/config_mapper/config_mapper.py [hunk 2]
```diff
@@ -76,34 +76,39 @@ def extract_root_dirs(config_obj: config_pb2.Directive) -> tuple:
...
     # these dicts can be updated when more plugins are supported
-    for d in config_obj.directives:
-        if d.name not in plugin_prefix_map:
+    for directive in config_obj.directives:
+        if directive.name not in plugin_prefix_map:
             stats['entities_skipped'] += 1
-            stats['attributes_skipped'] += _get_aggregated_num_attributes(d)
+            stats['attributes_skipped'] += _get_aggregated_num_attributes(
+                directive)
+            stats['warning_logs'] += 1
```
Comment (StevenYCChou) [inline config_converter/config_mapper/config_mapper.py:84]: I think increasing the stats after the logging statement would be more intuitive. I meant:
```
logging.warning(
                'Skip mapping %s due to missing functionality in master agent',
                directive.name)
stats['warning_logs'] += 1
```

It applies to all the log-related stats.

File: config_converter/config_mapper/config_mapper.py (hunk 1 above)
Comment (mihikabairathi) [reply config_converter/config_mapper/config_mapper.py:109]: You're right - I can't fit it on this line so I've put it on the line above. Do you think that's okay?

File: config_converter/config_mapper/config_mapper.py (hunk 2 above)
Comment (mihikabairathi) [reply config_converter/config_mapper/config_mapper.py:84]: Oh you're right it's much more clearer this way - I will change it everywhere!

**Pull request merged**